### PR TITLE
Bump iron 0.6.0

### PR DESCRIPTION
**Pull request by ernestas-poskus**
Depdends on urlencdoded 0.6.0: https://github.com/iron/urlencoded/pull/72

### CODE REVIEW

**Comment by untitaker** [comment]
Restarted build

**Comment by untitaker** [comment]
Build still breaks after urlencoded release, please fix

**Comment by jebrosen** [comment]
I was able to fix the errors locally by changing both instances of `[]` in src/request.rs to `b""`. I can't for the life of me understand what changed that brought out that error message.

**Comment by ernestas-poskus** [comment]
Fixed tests

File: Cargo.toml [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-name = "iron-test"
-version = "0.5.0"
 authors = ["Jonathan Reem <jonathan.reem@gmail.com>"]
 description = "Mocking suite for Iron requests."
-repository = "https://github.com/reem/iron-test"
 documentation = "https://docs.rs/iron-test"
 license = "MIT"
+name = "iron-test"
+repository = "https://github.com/reem/iron-test"
+version = "0.5.0"
```
Comment (robinst) [inline Cargo.toml]: Can you bump this to 0.6.0? Then we can release it after merging.

Comment (ernestas-poskus) [reply Cargo.toml]: done

**Comment by ernestas-poskus** [comment]
Bumped

**Comment by stepankuzmin** [comment]
Any news on that?

**Comment by robinst** [comment]
@SkylerLipthay Can you merge and release this one as well? Thanks!

**Comment by SkylerLipthay** [comment]
So unfortunately I think only @reem can publish this one? The iron team doesn't seem to have [crates.io permissions](https://crates.io/crates/iron-test).

**Comment by robinst** [comment]
@reem can you please run `cargo owner --add github:iron:dev-team` in `iron-test`?

**Comment by jebrosen** [comment]
The ownership appears to be fixed, any word on this issue?

**Comment by SkylerLipthay** [comment]
OK, `v0.6.0` has been published! Thanks for the patience everyone.

**Pull request merged**